### ISSUE TITLE
Allow validators to coerce types

### ISSUE THREAD

**Issue by ahernandez2-rbi** [samchon/typia#636]
## Feature Request
I'm trying to integrate Typia into Fastify validation and I'm running into an issue when trying to validate a querystring because the object receives all properties as strings:

```ts
type Car = { brand: string; year: number };

const validator = typia.createValidate<Car>();

validator({brand: 'nissan', year: '1000'}); // fails
```

Generated code looks like this:

```ts
const __is = input => {
    return "object" === typeof input && null !== input && ("number" === typeof input.year && "string" === typeof input.brand);
};
```

It would be ideal if we could have an option to coerce string to a number. This is something that the Fastify built-in Ajv integration supports.

**Comment by samchon** [comment]
https://nestia.io/docs/core/TypedQuery/

I'd also considered same issue long time ago, but decided that this issue is not suitable to purpose of `typia`. Instead, I've made a new project `nestia`, which can use `typia`'s validators in NestJS. If you're interested in the NestJS, what about using `TypedQuery` of `nestia`? It automatically converts types as you want.

For reference, Fastify team member (@metcoder95) had requested me to develop a plugin for Fastify. To respond to the request, at first, I made Nestia to support Fastify. When you adapt Nestia with Fastify, it boosts up NestJS backend server performance about 10~30x times higher. The next step is to just making pure plugin only for Fastify without wrapper like NestJS. I'm planning to start it at the end of next month, and if you have a good idea for interface designing, I'll listen you with pleasure.

![Performance Benchmark](https://raw.githubusercontent.com/samchon/nestia/master/benchmark/results/11th%20Gen%20Intel(R)%20Core(TM)%20i5-1135G7%20%40%202.40GHz/images/performance.svg)
